api/src/endpoints/neo4j.js

```diff
@@ -191,11 +223,16 @@ neo4jRoutes.get('/identifier/:dbName/:externalId', async (req, res) => {
   const { referenceType } = req.query;
 
   try {
+    validateInput(dbName);
+    validateInput(externalId);
+    validateInput(referenceType);
```

Comment:
This looks a bit ugly. What about an iterative control statement?

Reply:
Hmm...it may be a matter of taste but I do not see what is ugly about it. How would you like to re-write it?

Reply:
I was thinking of iterating using forEach. But if you think it's fine then OK!